src/old-api/tag.ts

```diff
@@ -22,4 +23,36 @@ export class Tag extends SpecificationExtensionsModel<v2.TagObject> {
   hasExternalDocs() {
     return hasExternalDocs(this);
   }
+
+  hasExtensions() {
```

Comment:
I had to do that due to circular references between tag, externalDocs and mixins files.